### PR TITLE
tweak: add end command

### PR DESCRIPTION
**Pull request by kien-ngo**

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -167,6 +167,43 @@ client.on('messageCreate', async (message) => {
...
+						// collect tags and add close tag
+						let initialTags = [closeTag[0].id,...postTags];
+						let tags = [...new Set(initialTags)];
+
+						// send embed message upon executing the close command
+						await message.channel.send({ 
+							embeds: [
+								sendEmbedMessage(`${config.reminder_close}`)
+							],
+							content: `🔔`
```
Comment (warengonzaga) [inline src/bot.js]: No longer needed. Let's remove the `content`

File: src/config.json [hunk 2]
```diff
@@ -2,10 +2,12 @@
     "command_prefix": "!",
     "command_resolve": "resolve",
     "command_close": "close",
+    "command_end": "end",
     "command_escalate": "escalate",
     "command_bug": "bug",
     "command_sc_resolve": "res",
     "command_sc_close": "cls",
+    "command_sc_close_no_mention": "!cls",
```
Comment (warengonzaga) [inline src/config.json]: Is this still visible from the code base? I don't see this. @kien-ngo

**Review by warengonzaga — CHANGES_REQUESTED** [review]
Overall, it is good; kindly check the comments from your changes. @kien-ngo

File: src/config.json (hunk 2 above)
Comment (kien-ngo) [reply src/config.json]: It was mistakenly added and I removed it. You can view all the changes here: https://github.com/thirdweb-dev/support-discord-bot/pull/68/files

File: src/bot.js (hunk 1 above)
Comment (kien-ngo) [reply src/bot.js]: @warengonzaga we can safely remove it right?

Comment (kien-ngo) [reply src/bot.js]: https://github.com/thirdweb-dev/support-discord-bot/pull/68/commits/1c7c24a20af24e4f1fa24ed95fd1a577cea6b889

Comment (warengonzaga) [reply src/bot.js]: yes, it is safe to remove.

**Review by warengonzaga — APPROVED** [review]
LGTM!

**Pull request merged**